pkgs/top-level/python-packages.nix

```diff
@@ -2696,6 +2696,7 @@ in {
   googleapis_common_protos = callPackage ../development/python-modules/googleapis_common_protos { };
 
   google-auth-httplib2 = callPackage ../development/python-modules/google-auth-httplib2 { };
+  google-auth-oauthlib = callPackage ../development/python-modules/google-auth-oauthlib { };
```

Comment:
Please add a blank line.

Reply:
Sorry, should have figured from looking at all the other ones.